src/main/views/user-details.njk

```diff
@@ -165,8 +165,12 @@
           {{ govukButton({ text: "Unsuspend user", name: "_action", value: "unsuspend", classes: "govuk-button--secondary" }) if not content.user.active }}
         {% endif %}
 
+        {% if featureFlags.values[featureFlags.flags.BETA_SSO] and (content.user.ssoProvider or content.user.ssoId) and content.showManage %}
+          {{ govukButton({ text: "Disable SSO", name: "_action", value: "disable-sso", classes: "govuk-button--secondary" })}}
```

Comment:
Think the ticket says it should be call 'Remove SSO' and not 'Disable SSO'. This endpoint is to remove the SSO info from the user

Reply:
fixed